src/main/java/hec/JdbcTimeSeriesDatabase.java

```diff
@@ -23,6 +23,11 @@
  */
 public class JdbcTimeSeriesDatabase extends TimeSeriesDatabase {
 
+    public enum CREATION_MODE {
```

Comment:
is this unique to jdbc?

Reply:
It's more specific to SQLite but we called it JDBC. If we were use library forfor say, the  CWMS oracle database, there wouldn't be any option except open the existing.